packages/shared/src/hooks/useInputFieldFunctions.ts

```diff
@@ -40,8 +40,10 @@ function useInputFieldFunctions<
   const [inputLength, setInputLength] = useState<number>(undefined);
   const [validInput, setValidInput] = useState<boolean>(undefined);
   const [idleTimeout, clearIdleTimeout] = useDebounceFn(() => {
-    setValidInput(inputRef.current.checkValidity());
-  }, 1500);
+    if (inputRef.current) {
+      setValidInput(inputRef.current.checkValidity());
+    }
+  }, 500);
```

Comment:
I have changed this to 500 because I noticed when the URL is invalid the error message below the input box shows a bit later and feels a little delayed.  Maybe you can test it and let me know if I should keep this 500 or 1500 same as before.